packages/e2e/src/scripts/generate-dotenv.sh

```diff
@@ -16,19 +26,24 @@ KEY_MANAGEMENT_PROVIDER=inMemory
 
 # Providers setup - required by getWallet
 ASSET_PROVIDER=http
-ASSET_PROVIDER_PARAMS='{\"baseUrl\":\"$url:4000/\"}'
+ASSET_PROVIDER_PARAMS='{\"baseUrl\":\"${url}\"}'
+CHAIN_HISTORY_PROVIDER=http
+CHAIN_HISTORY_PROVIDER_PARAMS='{\"baseUrl\":\"${url}\"}'
 HANDLE_PROVIDER=http
-HANDLE_PROVIDER_PARAMS='{\"baseUrl\":\"$url:4011/\"}'
+HANDLE_PROVIDER_PARAMS='{\"baseUrl\":\"${url}\"}'
 NETWORK_INFO_PROVIDER=http
-NETWORK_INFO_PROVIDER_PARAMS='{\"baseUrl\":\"$url:4000/\"}'
+NETWORK_INFO_PROVIDER_PARAMS='{\"baseUrl\":\"${url}\"}'
 REWARDS_PROVIDER=http
-REWARDS_PROVIDER_PARAMS='{\"baseUrl\":\"$url:4000/\"}'
+REWARDS_PROVIDER_PARAMS='{\"baseUrl\":\"${url}\"}'
 TX_SUBMIT_PROVIDER=http
-TX_SUBMIT_PROVIDER_PARAMS='{\"baseUrl\":\"$url:4000/\"}'
+TX_SUBMIT_PROVIDER_PARAMS='{\"baseUrl\":\"${url}\"}'
 UTXO_PROVIDER=http
-UTXO_PROVIDER_PARAMS='{\"baseUrl\":\"$url:4000/\"}'
+UTXO_PROVIDER_PARAMS='{\"baseUrl\":\"${url}\"}'
 STAKE_POOL_PROVIDER=http
-STAKE_POOL_PROVIDER_PARAMS='{\"baseUrl\":\"$url:4000/\"}'
+STAKE_POOL_PROVIDER_PARAMS='{\"baseUrl\":\"${url}\"}'
+WS_PROVIDER_URL='wss://${domain}/ws'
+
+PRE_CONWAY=stable
```

Comment:
We can exclude `PRE_CONWAY`. It is used to define the local-network docker image for babbage or conway era
```diff
- PRE_CONWAY=stable
```

Reply:
Thank you: TIL.

aff9269c37f should solve this